sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -44,7 +45,7 @@ def session_configs():
 
 @_with_override
 def kernel_python_credentials():
-    return {u'username': u'', u'base64_password': u'', u'url': u'http://localhost:8998'}
+    return {u'username': u'', u'base64_password': u'', u'url': u'http://localhost:8998', u'auth_type': constants.NO_AUTH}
```

Comment:
For this to be back compatible with old config.json files, we need to move the logic of:

`constants.NO_AUTH if username == '' and password == '' else constants.BASIC_AUTH`